src/main/kotlin/com/exactpro/th2/common/utils/message/transport/MessageUtils.kt

```diff
@@ -35,6 +35,12 @@ import com.exactpro.th2.common.value.toValue
 import java.math.BigDecimal
 import com.exactpro.th2.common.grpc.Message as ProtoMessage
 
+fun Message<*>.toBatch(book: String, sessionGroup: String): GroupBatch = GroupBatch.builder().apply {
+    setBook(book)
+    setSessionGroup(sessionGroup)
+    addGroup(this@toBatch.toGroup())
+}.build()
```

Comment:
We probably need to make this extension for `MessageGroup` instead of `Message<*>`. In this case, we will be able to call `message.toGroup().toBatch("book", "group")` and `group.toBatch("book", "group")`.

Right now we cannot create batch from group as I can see